scripts/release_info.py

```diff
@@ -51,16 +56,29 @@
                     'CURRENT_COLLECTION_SCHEMA_VERSION']
 FIRST_OPPIA_COMMIT = '6a7138f5f603375e58d1dc3e1c4f1c80a126e249'
 NO_LABEL_CHANGELOG_CATEGORY = 'Uncategorized'
+# This should match the indentation of span and li elements used
+# in credits section of
+# core/templates/dev/head/pages/about-page/about-page.directive.html.
+SPAN_INDENT = '              '
+LI_INDENT = '                '
+# This line should match the line after div element for credits in
```

Comment:
Same thing, try and test this to ensure that no incoming PR can break this script's functionality.

Reply:
Replied above.

Reply:
Done.